Exercises/functions/functions.js

```diff
@@ -1,3 +1,31 @@
-function takeMyParametersAmount(a, b, c) {
+// Task #1
+function functionDeclaration(a, b, ...rest) {
   return null;
 }
+
+const functionExpression = function (a, b, ...rest) { return null; };
+
+const functionObject = new Function ('a', 'b', 'return null');
+
+const arrowFunction = (a, b, ...rest) => null;
+
+// Task #2
+function takeMyParametersAmount() {
+  let i = 0;
+  for (let arg of arguments)
+    i += 1;
+  console.log(i);
+  return null;
+}
+takeMyParametersAmount('a','b','c');
+
+// Task #3
+function logPassedParams() {
+  return arguments.length;
+}
```

Comment:
Rest syntax should be preferred:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters
Also you log not params, but only their length